examples/resource_lacework_integration_ghcr/variables.tf

```
@@ -13,4 +13,7 @@ variable "password" {
 variable "ssl" {
   type = bool
 }
-
+variable "non_os_packages" {
```

Please update the rest and format all Terraform files
```suggestion
variable "non_os_package_support" {
```